cpp/ps0002/ps0002_solv.cpp

```diff
@@ -4,15 +4,17 @@ using namespace std;
 
 int main()
 {
-    unsigned int calc_neighbors =500;
-    int Alternative_medicines =1000;
-    float piece_bread= 12.5;
-    char first_letter='A';
-    unsigned long int calc_China_Births=1234567890;
-    int division=2;
-    long double calc_Electricity_production=12345.1;
-    unsigned long long int Age_seconds=11223344556677;
-    cout <<calc_neighbors<<"\n" <<Alternative_medicines<<"\n" <<piece_bread<<"\n" <<first_letter<<"\n"
-    <<calc_China_Births<<"\n" <<division<<"\n" <<calc_Electricity_production<<"\n" <<Age_seconds<<"\n" ;
+    unsigned short calc_neighbors = 500; // 0.5/0.75
+    unsigned short alternative_medicines = 1000; // 0.75/0.75
+    unsigned short team_members = 4; // 0.75/0.75
+    float piece_bread = 12.5; // 0.75/0.75
+    char first_letter = 'A'; // 0.75/0.75
+    unsigned int calc_China_Births = 1234567890; // 0.5/0.75
+    long double division = 2.565477654; // 0.25/0.75
+    float calc_Electricity_production = 12345.1; // 0.5/0.75
+    unsigned int Age_seconds = 1144556677; // 0.5/0.75 
+    // total 5.25/6.75
+    
     return 0;
 }
+// good work 👍 7.5/10

```

Comment:
- [x] program is in the correct path
- [x] no syntax errors
- [x] formatted code to be clear and readable
- [ ] commented some phrases for blocks of code